daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -146,6 +146,11 @@ function Configure-Network {
   # Register netkvmco.dll.
   Run-Command rundll32 'netkvmco.dll,RegisterNetKVMNetShHelper'
 
+  # Remove ReadOnly file attribute from host_file, if present.
```

Comment:
Might I also share an alternative way which is to use the `-Force` with `Add-Content` [Ref: [Add-Content doc](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/add-content?view=powershell-5.1)]? It might be simpler this way. It also doesn't alter the original file permission. Thanks. 

```powershell
Write-Host "Appending Google Compute Engine metadata server address to host file $($script:hosts_file)."
Add-Content -Path $script:hosts_file -Value "# Google Compute Engine metadata server`n    169.254.169.254    metadata.google.internal metadata" -Force
```

Reply:
Thanks, I like that simplier solution.